persistence/subsystem/eventstore/repository.go

```diff
@@ -9,6 +9,16 @@ type Repository interface {
 	// NextEventOffset returns the next "unused" offset within the store.
 	NextEventOffset(ctx context.Context) (uint64, error)
 
+	// LoadEventsBySource loads the events produced by the specified source with
+	// the given handler key and id.
+	//
+	// d is the optional parameter, it represents ID of the message that was
+	// recorded when the instance was last destroyed.
```

Comment:
```suggestion
	// LoadEventsBySource loads the events produced by a specific handler.
	//
	// hk is the handler's identity key.
	//
	// id is the instance ID, which must be empty if the handler type does not
	// use instances.
	//
	// m is ID of a "barrier" message. If supplied, the results are limited to
	// events with higher offsets than the barrier message.
```

Since we decided to make this `LoadEventsBySource()`, rather than `LoadEventsForAggregate()`, I felt that the `d` parameter was too aggregate-specific, so I've tried to reword the docs just to "keep this layer dumb". Nothing I've written there changes the necessary behavior in any way.